po/en-US/taidan.ftl

```diff
@@ -170,22 +170,22 @@ categories = { $cat }
 page-installing = Installing Your Apps
 page-installing-desc = This won't take long!
 page-installing-loading = Loading…
-page-installing-flatpak = Installing {$n} flatpaks…
+page-installing-flatpak = Installing {$n} Flatpaks…
 
 ## page: Finish
 
 # .done: btn; closes the entire application
-page-finish = Your System is Ready
-page-finish-desc = We hope you enjoy your fresh system!
+page-finish = Ready to Do Your Thing?
+page-finish-desc = We hope you enjoy Ultramarine!
```

Comment:
can you use `{ $distro }` instead? (expands to `Ultramarine Linux`)

I'll add the code for the substitution in another commit.